compendium/DeclarativeServices/src/manager/ComponentConfigurationImpl.cpp

```diff
@@ -488,18 +489,24 @@ namespace cppmicroservices
         InstanceContextPair
         ComponentConfigurationImpl::CreateAndActivateComponentInstanceHelper(cppmicroservices::Bundle const& bundle)
         {
+            static Guarded<std::unordered_set<std::string>> bundleValidated;
```

Comment:
this is somewhat redundant to `static Guarded<std::map<std::string, void*>> bundleBinaries;` used in `GetComponentCreatorDeletors`. Do we need both maps or can this be consolidated?